allensdk/brain_observatory/behavior/behavior_project_cache/tables/util/metadata_parsers.py

```diff
@@ -0,0 +1,245 @@
+from typing import Union
+
+import pandas as pd
+
+#####################################################################
+#
+#                   VBO PROJECT CONSTANTS
+#
+#####################################################################
+
+
+SESSION_TYPES = [
```

Comment:
I don't think you need to define `SESSION_TYPES` and `PROJECT_CODES`. Just try to parse and raise exception if unable to parse